code/game/atoms_movable.dm

```diff
@@ -754,4 +754,92 @@
 	appearance = falltempoverlays[src]
 	falltempoverlays -= src
 	ignoreinvert = initial(ignoreinvert)
-	invisibility = init_invisibility
\ No newline at end of file
+	invisibility = init_invisibility
+
+/datum/proc/send_to_past(var/duration)
+	return
+
+/datum/var/being_sent_to_past = FALSE
+
+/atom/movable/send_to_past(var/duration)
+	var/current_loc = loc
+	var/list/resettable_vars = list(
+									"being_sent_to_past",
```

Comment:
Not sure why Github did that, looks fine locally.
![image](https://cloud.githubusercontent.com/assets/3991927/22498016/4089a8f6-e81b-11e6-9b2b-3bb2afac3a76.png)